lib/features/component/editText/stark_amount.dart

```diff
@@ -0,0 +1,68 @@
+part of '../../feature.dart';
+
+class StarkAmount extends StatelessWidget {
```

Comment:
this is not an editText so it can't be in the editText folder. you can just put it in the component folder directly 

Also this is similar with what's already in the home screen. use the same widget in the home screen